client/src/pages/DriverLocationInfo/DriverLocationInfo.tsx

```diff
@@ -209,6 +210,12 @@ export function DriverLocationInfo() {
   ) {
     const error =
       driversInfoQuery.error || dropoffLocationsQuery.error || eventQuery.error;
+    if (error instanceof Error && error.message.includes("token")) {
+      setToken(null);
+      localStorage.removeItem("token");
+      toastNotify("Sorry, please login again");
+      return <Navigate to="/" />;
+    }
```

Comment:
I see that this is working and was able to see it work in the test deployment. I got the error and re-authenticated, so it's definitely working.
I see that the code is implemented in each page pretty much, for Events, View Events, and Driver info pages. Just curious if there's any way of doing that more centrally so that we don't have to handle for every new type of page on its own. More of a question for ideal architecture, i'll approve this one to address the needed gap that it fixes now.

Reply:
Yeah I debated making a `handleTokenExpiration` function, but since it's only four lines of code, I didn't think it was a big deal. Additionally, we only have three pages in the app, so this check is only necessary in a few places. If it becomes a problem in the future, we can rework it.

Reply:
makes sense.